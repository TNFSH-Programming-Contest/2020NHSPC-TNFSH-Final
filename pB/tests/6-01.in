874
-907.844 341.145
-910.273 334.61
-912.655 328.057
-914.99 321.488
-917.277 314.902
-919.517 308.299
-921.71 301.681
-923.855 295.047
-925.952 288.398
-928.002 281.734
-930.003 275.055
-931.956 268.362
-933.861 261.656
-935.718 254.935
-937.527 248.202
-939.287 241.456
-940.998 234.697
-942.661 227.926
-944.276 221.144
-945.841 214.35
-947.357 207.544
-948.825 200.729
-950.243 193.902
-951.613 187.066
-952.933 180.22
-954.204 173.365
-955.426 166.501
-956.598 159.628
-957.721 152.747
-958.794 145.858
-959.818 138.962
-960.792 132.058
-961.717 125.147
-962.592 118.23
-963.417 111.307
-964.192 104.379
-964.917 97.444
-965.593 90.505
-966.219 83.561
-966.794 76.613
-967.32 69.661
-967.796 62.705
-968.222 55.746
-968.597 48.784
-968.923 41.819
-969.199 34.853
-969.424 27.884
-969.6 20.915
-969.725 13.944
-969.8 6.972
-969.825 0.0
-969.8 -6.972
-969.725 -13.944
-969.6 -20.915
-969.424 -27.884
-969.199 -34.853
-968.923 -41.819
-968.597 -48.784
-968.222 -55.746
-967.796 -62.705
-967.32 -69.661
-966.794 -76.613
-966.219 -83.561
-965.593 -90.505
-964.917 -97.444
-964.192 -104.379
-963.417 -111.307
-962.592 -118.23
-961.717 -125.147
-960.792 -132.058
-959.818 -138.962
-958.794 -145.858
-957.721 -152.747
-956.598 -159.628
-955.426 -166.501
-954.204 -173.365
-952.933 -180.22
-951.613 -187.066
-950.243 -193.902
-948.825 -200.729
-947.357 -207.544
-945.841 -214.35
-944.276 -221.144
-942.661 -227.926
-940.998 -234.697
-939.287 -241.456
-937.527 -248.202
-935.718 -254.935
-933.861 -261.656
-931.956 -268.362
-930.003 -275.055
-928.002 -281.734
-925.952 -288.398
-923.855 -295.047
-921.71 -301.681
-919.517 -308.299
-917.277 -314.902
-914.99 -321.488
-912.655 -328.057
-910.273 -334.61
-907.844 -341.145
-905.368 -347.663
-902.845 -354.162
-900.276 -360.644
-897.66 -367.106
-894.998 -373.55
-892.289 -379.975
-889.535 -386.379
-886.734 -392.764
-883.887 -399.129
-880.995 -405.473
-878.058 -411.796
-875.075 -418.097
-872.046 -424.377
-868.973 -430.635
-865.855 -436.871
-862.692 -443.085
-859.484 -449.275
-856.232 -455.442
-852.936 -461.586
-849.595 -467.706
-846.211 -473.801
-842.783 -479.872
-839.312 -485.919
-835.797 -491.94
-832.239 -497.936
-828.637 -503.906
-824.993 -509.85
-821.307 -515.767
-817.578 -521.658
-813.807 -527.522
-809.993 -533.359
-806.138 -539.168
-802.241 -544.95
-798.303 -550.703
-794.323 -556.427
-790.303 -562.123
-786.241 -567.79
-782.139 -573.428
-777.996 -579.036
-773.814 -584.614
-769.591 -590.162
-765.328 -595.679
-761.026 -601.165
-756.685 -606.621
-752.304 -612.045
-747.885 -617.437
-743.427 -622.798
-738.93 -628.126
-734.396 -633.422
-729.823 -638.685
-725.213 -643.916
-720.565 -649.112
-715.88 -654.276
-711.158 -659.405
-706.399 -664.501
-701.604 -669.562
-696.772 -674.588
-691.905 -679.58
-687.001 -684.536
-682.062 -689.457
-677.088 -694.343
-672.079 -699.193
-667.035 -704.006
-661.957 -708.783
-656.845 -713.524
-651.698 -718.227
-646.518 -722.894
-641.305 -727.523
-636.058 -732.114
-630.778 -736.668
-625.466 -741.183
-620.122 -745.661
-614.745 -750.099
-609.337 -754.499
-603.897 -758.86
-598.426 -763.182
-592.924 -767.465
-587.392 -771.707
-581.829 -775.91
-576.236 -780.073
-570.613 -784.195
-564.961 -788.277
-559.279 -792.318
-553.569 -796.318
-547.83 -800.277
-542.062 -804.195
-536.267 -808.071
-530.444 -811.905
-524.594 -815.697
-518.716 -819.448
-512.812 -823.155
-506.881 -826.821
-500.924 -830.443
-494.941 -834.023
-488.932 -837.56
-482.899 -841.053
-476.84 -844.503
-470.756 -847.909
-464.649 -851.271
-458.517 -854.589
-452.361 -857.864
-446.183 -861.093
-439.981 -864.279
-433.756 -867.419
-427.509 -870.515
-421.24 -873.566
-414.949 -876.572
-408.637 -879.532
-402.303 -882.447
-395.949 -885.316
-389.574 -888.14
-383.179 -890.918
-376.765 -893.649
-370.331 -896.335
-363.877 -898.974
-357.405 -901.567
-350.915 -904.113
-344.406 -906.612
-337.88 -909.064
-331.336 -911.47
-324.775 -913.828
-318.197 -916.14
-311.603 -918.403
-304.992 -920.62
-298.366 -922.788
-291.724 -924.91
-285.068 -926.983
-278.396 -929.008
-271.711 -930.986
-265.011 -932.915
-258.297 -934.796
-251.57 -936.629
-244.831 -938.413
-238.078 -940.149
-231.313 -941.836
-224.536 -943.475
-217.748 -945.064
-210.948 -946.605
-204.138 -948.097
-197.317 -949.54
-190.485 -950.934
-183.644 -952.279
-176.794 -953.575
-169.934 -954.821
-163.065 -956.018
-156.188 -957.166
-149.303 -958.264
-142.411 -959.312
-135.511 -960.311
-128.603 -961.261
-121.69 -962.16
-114.77 -963.01
-107.844 -963.81
-100.912 -964.561
-93.975 -965.261
-87.034 -965.912
-80.088 -966.513
-73.137 -967.063
-66.183 -967.564
-59.226 -968.015
-52.265 -968.416
-45.302 -968.767
-38.336 -969.067
-31.369 -969.318
-24.4 -969.518
-17.429 -969.669
-10.458 -969.769
-3.486 -969.819
3.486 -969.819
10.458 -969.769
17.429 -969.669
24.4 -969.518
31.369 -969.318
38.336 -969.067
45.302 -968.767
52.265 -968.416
59.226 -968.015
66.183 -967.564
73.137 -967.063
80.088 -966.513
87.034 -965.912
93.975 -965.261
100.912 -964.561
107.844 -963.81
114.77 -963.01
121.69 -962.16
128.603 -961.261
135.511 -960.311
142.411 -959.312
149.303 -958.264
156.188 -957.166
163.065 -956.018
169.934 -954.821
176.794 -953.575
183.644 -952.279
190.485 -950.934
197.317 -949.54
204.138 -948.097
210.948 -946.605
217.748 -945.064
224.536 -943.475
231.313 -941.836
238.078 -940.149
244.831 -938.413
251.57 -936.629
258.297 -934.796
265.011 -932.915
271.711 -930.986
278.396 -929.008
285.068 -926.983
291.724 -924.91
298.366 -922.788
304.992 -920.62
311.603 -918.403
318.197 -916.14
324.775 -913.828
331.336 -911.47
337.88 -909.064
344.406 -906.612
350.915 -904.113
357.405 -901.567
363.877 -898.974
370.331 -896.335
376.765 -893.649
383.179 -890.918
389.574 -888.14
395.949 -885.316
402.303 -882.447
408.637 -879.532
414.949 -876.572
421.24 -873.566
427.509 -870.515
433.756 -867.419
439.981 -864.279
446.183 -861.093
452.361 -857.864
458.517 -854.589
464.649 -851.271
470.756 -847.909
476.84 -844.503
482.899 -841.053
488.932 -837.56
494.941 -834.023
500.924 -830.443
506.881 -826.821
512.812 -823.155
518.716 -819.448
524.594 -815.697
530.444 -811.905
536.267 -808.071
542.062 -804.195
547.83 -800.277
553.569 -796.318
559.279 -792.318
564.961 -788.277
570.613 -784.195
576.236 -780.073
581.829 -775.91
587.392 -771.707
592.924 -767.465
598.426 -763.182
603.897 -758.86
609.337 -754.499
614.745 -750.099
620.122 -745.661
625.466 -741.183
630.778 -736.668
636.058 -732.114
641.305 -727.523
646.518 -722.894
651.698 -718.227
656.845 -713.524
661.957 -708.783
667.035 -704.006
672.079 -699.193
677.088 -694.343
682.062 -689.457
687.001 -684.536
691.905 -679.58
696.772 -674.588
701.604 -669.562
706.399 -664.501
711.158 -659.405
715.88 -654.276
720.565 -649.112
725.213 -643.916
729.823 -638.685
734.396 -633.422
738.93 -628.126
743.427 -622.798
747.885 -617.437
752.304 -612.045
756.685 -606.621
761.026 -601.165
765.328 -595.679
769.591 -590.162
773.814 -584.614
777.996 -579.036
782.139 -573.428
786.241 -567.79
790.303 -562.123
794.323 -556.427
798.303 -550.703
802.241 -544.95
806.138 -539.168
809.993 -533.359
813.807 -527.522
817.578 -521.658
821.307 -515.767
824.993 -509.85
828.637 -503.906
832.239 -497.936
835.797 -491.94
839.312 -485.919
842.783 -479.872
846.211 -473.801
849.595 -467.706
852.936 -461.586
856.232 -455.442
859.484 -449.275
862.692 -443.085
865.855 -436.871
868.973 -430.635
872.046 -424.377
875.075 -418.097
878.058 -411.796
880.995 -405.473
883.887 -399.129
886.734 -392.764
889.535 -386.379
892.289 -379.975
894.998 -373.55
897.66 -367.106
900.276 -360.644
902.845 -354.162
905.368 -347.663
907.844 -341.145
910.273 -334.61
912.655 -328.057
914.99 -321.488
917.277 -314.902
919.517 -308.299
921.71 -301.681
923.855 -295.047
925.952 -288.398
928.002 -281.734
930.003 -275.055
931.956 -268.362
933.861 -261.656
935.718 -254.935
937.527 -248.202
939.287 -241.456
940.998 -234.697
942.661 -227.926
944.276 -221.144
945.841 -214.35
947.357 -207.544
948.825 -200.729
950.243 -193.902
951.613 -187.066
952.933 -180.22
954.204 -173.365
955.426 -166.501
956.598 -159.628
957.721 -152.747
958.794 -145.858
959.818 -138.962
960.792 -132.058
961.717 -125.147
962.592 -118.23
963.417 -111.307
964.192 -104.379
964.917 -97.444
965.593 -90.505
966.219 -83.561
966.794 -76.613
967.32 -69.661
967.796 -62.705
968.222 -55.746
968.597 -48.784
968.923 -41.819
969.199 -34.853
969.424 -27.884
969.6 -20.915
969.725 -13.944
969.8 -6.972
969.825 0.0
969.8 6.972
969.725 13.944
969.6 20.915
969.424 27.884
969.199 34.853
968.923 41.819
968.597 48.784
968.222 55.746
967.796 62.705
967.32 69.661
966.794 76.613
966.219 83.561
965.593 90.505
964.917 97.444
964.192 104.379
963.417 111.307
962.592 118.23
961.717 125.147
960.792 132.058
959.818 138.962
958.794 145.858
957.721 152.747
956.598 159.628
955.426 166.501
954.204 173.365
952.933 180.22
951.613 187.066
950.243 193.902
948.825 200.729
947.357 207.544
945.841 214.35
944.276 221.144
942.661 227.926
940.998 234.697
939.287 241.456
937.527 248.202
935.718 254.935
933.861 261.656
931.956 268.362
930.003 275.055
928.002 281.734
925.952 288.398
923.855 295.047
921.71 301.681
919.517 308.299
917.277 314.902
914.99 321.488
912.655 328.057
910.273 334.61
907.844 341.145
905.368 347.663
902.845 354.162
900.276 360.644
897.66 367.106
894.998 373.55
892.289 379.975
889.535 386.379
886.734 392.764
883.887 399.129
880.995 405.473
878.058 411.796
875.075 418.097
872.046 424.377
868.973 430.635
865.855 436.871
862.692 443.085
859.484 449.275
856.232 455.442
852.936 461.586
849.595 467.706
846.211 473.801
842.783 479.872
839.312 485.919
835.797 491.94
832.239 497.936
828.637 503.906
824.993 509.85
821.307 515.767
817.578 521.658
813.807 527.522
809.993 533.359
806.138 539.168
802.241 544.95
798.303 550.703
794.323 556.427
790.303 562.123
786.241 567.79
782.139 573.428
777.996 579.036
773.814 584.614
769.591 590.162
765.328 595.679
761.026 601.165
756.685 606.621
752.304 612.045
747.885 617.437
743.427 622.798
738.93 628.126
734.396 633.422
729.823 638.685
725.213 643.916
720.565 649.112
715.88 654.276
711.158 659.405
706.399 664.501
701.604 669.562
696.772 674.588
691.905 679.58
687.001 684.536
682.062 689.457
677.088 694.343
672.079 699.193
667.035 704.006
661.957 708.783
656.845 713.524
651.698 718.227
646.518 722.894
641.305 727.523
636.058 732.114
630.778 736.668
625.466 741.183
620.122 745.661
614.745 750.099
609.337 754.499
603.897 758.86
598.426 763.182
592.924 767.465
587.392 771.707
581.829 775.91
576.236 780.073
570.613 784.195
564.961 788.277
559.279 792.318
553.569 796.318
547.83 800.277
542.062 804.195
536.267 808.071
530.444 811.905
524.594 815.697
518.716 819.448
512.812 823.155
506.881 826.821
500.924 830.443
494.941 834.023
488.932 837.56
482.899 841.053
476.84 844.503
470.756 847.909
464.649 851.271
458.517 854.589
452.361 857.864
446.183 861.093
439.981 864.279
433.756 867.419
427.509 870.515
421.24 873.566
414.949 876.572
408.637 879.532
402.303 882.447
395.949 885.316
389.574 888.14
383.179 890.918
376.765 893.649
370.331 896.335
363.877 898.974
357.405 901.567
350.915 904.113
344.406 906.612
337.88 909.064
331.336 911.47
324.775 913.828
318.197 916.14
311.603 918.403
304.992 920.62
298.366 922.788
291.724 924.91
285.068 926.983
278.396 929.008
271.711 930.986
265.011 932.915
258.297 934.796
251.57 936.629
244.831 938.413
238.078 940.149
231.313 941.836
224.536 943.475
217.748 945.064
210.948 946.605
204.138 948.097
197.317 949.54
190.485 950.934
183.644 952.279
176.794 953.575
169.934 954.821
163.065 956.018
156.188 957.166
149.303 958.264
142.411 959.312
135.511 960.311
128.603 961.261
121.69 962.16
114.77 963.01
107.844 963.81
100.912 964.561
93.975 965.261
87.034 965.912
80.088 966.513
73.137 967.063
66.183 967.564
59.226 968.015
52.265 968.416
45.302 968.767
38.336 969.067
31.369 969.318
24.4 969.518
17.429 969.669
10.458 969.769
3.486 969.819
-3.486 969.819
-10.458 969.769
-17.429 969.669
-24.4 969.518
-31.369 969.318
-38.336 969.067
-45.302 968.767
-52.265 968.416
-59.226 968.015
-66.183 967.564
-73.137 967.063
-80.088 966.513
-87.034 965.912
-93.975 965.261
-100.912 964.561
-107.844 963.81
-114.77 963.01
-121.69 962.16
-128.603 961.261
-135.511 960.311
-142.411 959.312
-149.303 958.264
-156.188 957.166
-163.065 956.018
-169.934 954.821
-176.794 953.575
-183.644 952.279
-190.485 950.934
-197.317 949.54
-204.138 948.097
-210.948 946.605
-217.748 945.064
-224.536 943.475
-231.313 941.836
-238.078 940.149
-244.831 938.413
-251.57 936.629
-258.297 934.796
-265.011 932.915
-271.711 930.986
-278.396 929.008
-285.068 926.983
-291.724 924.91
-298.366 922.788
-304.992 920.62
-311.603 918.403
-318.197 916.14
-324.775 913.828
-331.336 911.47
-337.88 909.064
-344.406 906.612
-350.915 904.113
-357.405 901.567
-363.877 898.974
-370.331 896.335
-376.765 893.649
-383.179 890.918
-389.574 888.14
-395.949 885.316
-402.303 882.447
-408.637 879.532
-414.949 876.572
-421.24 873.566
-427.509 870.515
-433.756 867.419
-439.981 864.279
-446.183 861.093
-452.361 857.864
-458.517 854.589
-464.649 851.271
-470.756 847.909
-476.84 844.503
-482.899 841.053
-488.932 837.56
-494.941 834.023
-500.924 830.443
-506.881 826.821
-512.812 823.155
-518.716 819.448
-524.594 815.697
-530.444 811.905
-536.267 808.071
-542.062 804.195
-547.83 800.277
-553.569 796.318
-559.279 792.318
-564.961 788.277
-570.613 784.195
-576.236 780.073
-581.829 775.91
-587.392 771.707
-592.924 767.465
-598.426 763.182
-603.897 758.86
-609.337 754.499
-614.745 750.099
-620.122 745.661
-625.466 741.183
-630.778 736.668
-636.058 732.114
-641.305 727.523
-646.518 722.894
-651.698 718.227
-656.845 713.524
-661.957 708.783
-667.035 704.006
-672.079 699.193
-677.088 694.343
-682.062 689.457
-687.001 684.536
-691.905 679.58
-696.772 674.588
-701.604 669.562
-706.399 664.501
-711.158 659.405
-715.88 654.276
-720.565 649.112
-725.213 643.916
-729.823 638.685
-734.396 633.422
-738.93 628.126
-743.427 622.798
-747.885 617.437
-752.304 612.045
-756.685 606.621
-761.026 601.165
-765.328 595.679
-769.591 590.162
-773.814 584.614
-777.996 579.036
-782.139 573.428
-786.241 567.79
-790.303 562.123
-794.323 556.427
-798.303 550.703
-802.241 544.95
-806.138 539.168
-809.993 533.359
-813.807 527.522
-817.578 521.658
-821.307 515.767
-824.993 509.85
-828.637 503.906
-832.239 497.936
-835.797 491.94
-839.312 485.919
-842.783 479.872
-846.211 473.801
-849.595 467.706
-852.936 461.586
-856.232 455.442
-859.484 449.275
-862.692 443.085
-865.855 436.871
-868.973 430.635
-872.046 424.377
-875.075 418.097
-878.058 411.796
-880.995 405.473
-883.887 399.129
-886.734 392.764
-889.535 386.379
-892.289 379.975
-894.998 373.55
-897.66 367.106
-900.276 360.644
-902.845 354.162
-905.368 347.663
-390.769 -82.669
