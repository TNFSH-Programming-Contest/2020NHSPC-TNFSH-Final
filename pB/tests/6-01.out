5889941.194